envs/monkey_zoo/docs/zoo_setup.md

```diff
@@ -0,0 +1,140 @@
+
+        Compute Engine -> Compute Network Admin
+        Compute Engine -> Compute Instance Admin (v1)
+        Compute Engine -> Compute Security Admin
+        Service Account User
+
+    or just give
+
+       Project -> Owner
+       **Note: Adds more permissions then are needed**
+
+    c. Create and download its `Service account key` in JSON and place it in `monkey_zoo/gcp_keys` as `gcp_key.json`.
+
+2.  Get these permissions in our monkeyZoo production project (guardicore-22050661) for your service account.<br>
+    **Note: Ask monkey developers to add them. Check [Infection Monkey documentation](https://techdocs.akamai.com/infection-monkey/docs/welcome-infection-monkey) on how to recieve support**
+
+       Compute Engine -\> Compute image user
+
+3.  Change configurations located in the
+    ../monkey/envs/monkey\_zoo/terraform/config.tf file (don’t forget to
+    link to your service account key file):
+
+         provider "google" {
+
+         project = "test-000000" // Change to your project id
+
+           region  = "europe-west3" // Change to your desired region or leave default
+
+           zone    = "europe-west3-b" // Change to your desired zone or leave default
+
+           credentials = "${file("../gcp_keys/gcp_key.json")}" // Change to the location and name of the service key.
+                                                               // If you followed instruction above leave it as is
+
+         }
+
+         locals {
+
+           resource_prefix = "" // All of the resources will have this prefix.
+                                // Only change if you want to have multiple zoo's in the same project
+
+           service_account_email="tester-monkeyZoo-user@testproject-000000.iam.gserviceaccount.com" // Service account email
+
+           monkeyzoo_project="guardicore-22050661" // Project where monkeyzoo images are kept. Leave as is.
+
+         }
+
+4.  Run terraform init
+
+To deploy the network run:<br>
+`cd ../monkey/envs/monkey\_zoo/terraform`<br>
+`terraform plan` (review the changes it will make on GCP)<br>
+`terraform apply` (adds machines to these networks)
+
+
+## Using MonkeyZoo islands:
+
+### How to get into the islands:
+
+`island-linux-250`: SSH from GCP
+
+`island-windows-251`: In GCP/VM instances page click on
+`island-windows-251`. Set password for your account and then RDP into
+the island.
+
+You can find more information on MonkeyZoo machines in [MonkeyZoo network](zoo_network.md).
+
+
+## Installing Monkey Island in MonkeyZoo
+
+### For users
+
+#### Linux
+
+Upload the [`AppImage deployment`](https://github.com/guardicore/monkey/releases/latest)
+option and run it in [`island-linux-250`](zoo_network.md#_Toc536021489).
+
+#### Windows
+
+Upload the [`MSI deployment`](https://github.com/guardicore/monkey/releases/latest)
+option, install it and run it in [`island-windows-251`](zoo_network.md#_Toc536021490).
+After that use the Monkey as you would on local network.
```

Comment:
This whole section should probably be simplified to "Login to `island-linux-250` or `island-windows-251` depending on your preferred operating system. Then follow the [setup guide](https://techdocs.akamai.com/infection-monkey/docs/setting-up-infection-monkey)"